spec/features/comments_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Commenting', type: :feature do
+  given(:user) { FactoryGirl.create :user }
+  given(:poll) { FactoryGirl.create :valid_poll, :with_comments, user_id: user.id }
+
+  before(:each) { login_as(user) }
+
+  feature 'when add comment to poll' do
+    scenario 'have field for new comment' do
+      visit poll_path(poll)
+
+      expect(page).to have_field('comment_body')
+    end
+
+    scenario 'create comment', js: true do
+      create_comment('test comment text')
+
+      expect(page).to have_content('test comment text')
+    end
+  end
+
+  feature 'when add comment to another comment', js: true do
+    scenario 'comment have form for reply' do
+      visit poll_path(poll)
+
+      within "#comment_#{poll.comments.first.id}" do
+        expect(page).to have_link('Reply')
+        expect(page).to_not have_field('comment_body')
+
+        click_link 'Reply'
+
+        expect(page).to_not have_link('Reply')
```

Comment:
`expect(page).to have_no_link('Reply')` is actually waaay better — because it'll be waiting properly if Ajax not finished.

This is actually one of the best-kept secrets of reliable feature tests with Capybara, more on this [here](https://robots.thoughtbot.com/write-reliable-asynchronous-integration-tests-with-capybara#capybara-to-the-rescue) and [here](http://stefan.haflidason.com/testing-with-rails-and-capybara-methods-that-wait-method-that-wont/).

Please make sure that all your feature specs are following those advice, btw.